src/dataset-builder/CohortEditor.ts

```diff
@@ -573,12 +573,20 @@ interface CohortEditorProps {
   readonly snapshotBuilderSettings: SnapshotBuilderSettings;
   readonly originalCohort: Cohort;
   readonly updateCohorts: Updater<Cohort[]>;
+  readonly updateSelectedCohortsOnCreate: (cohort: Cohort) => void;
```

Comment:
Nit: How would you feel about just calling this `updateSelectedCohorts`? I am a bit concerned that `OnCreate` is a detail about the usage not about the prop itself. (Or maybe `addSelectedCohort`, actually? Since its only a single cohort?)

Reply:
Let me know what you think of the new names - they ended up getting reworked when I made the utils method more generic.